README.md

```diff
@@ -2,54 +2,61 @@
+Security: Properly configured access-control to resources/security groups, using secret storage systems (AWS Secret Manager, Kubernetes secrets), and following best practices provides great security out of the box. Our practices are built on top of multiple security audits and penetration tests. Automatic certificate management using [Let's Encrypt][letsencrypt], database encryption, VPN support, and more means your traffic will always be encrypted. Built-in application features like user authentication help you bullet-proof your application by using existing, tested tools rather than reinventing the wheel when it comes to features like user management and auth.
 
 
 ## What do you get out of the box?
+[Read about why we made these technology choices and where they are most applicable.](docs/technology-choices.md)
+
+[Check out some resources for learning more about these technologies.](docs/learning-resources.md)
+
 ### Infrastructure
 - Fully configured infrastructure-as-code AWS environment including:
   - VPCs per environment (staging, production) with pre-configured subnets, security groups, etc.
-  - EKS Kubernetes cluster per environment, pre-configured with tools like cert-manager, external-dns, nginx-ingress-controller
+  - EKS Kubernetes cluster per environment, pre-configured with helpful tools like cert-manager, external-dns, nginx-ingress-controller
   - RDS database for your application (Postgres or MySQL)
   - S3 buckets and Cloudfront distributions to serve your assets
 - Logging and Metrics collected automatically using either Cloudwatch or Prometheus + Grafana, Elasticsearch + Kibana
-- VPN using [Wireguard][wireguard]
+- VPN using [Wireguard][wireguard] (Optional)
 - User management and Identity / Access Proxy using Ory [Kratos][kratos] and [Oathkeeper][oathkeeper] (Optional)
-- Tooling to make it easy to set up secure access for your team
-- Local/Cloud Hybrid developer environment using Skaffold and Telepresence (In Progress)
+- Tooling to make it easy to set up secure access for your dev team
+- Local/Cloud Hybrid developer environment using Telepresence (Optional)
 
 ### Backend
 - Golang or Node.js example project automatically set up, Dockerized, and deployed to your new Kubernetes cluster
-- CI pipeline built with [CircleCI][circleci]. Just merge a PR and a deploy will start. Your code will be built and tested, deployed to staging, then prompt you to push to production
+- CI pipeline built with [CircleCI][circleci] or GitHubActions. Just merge a PR and a deploy will start. Your code will be built and tested, deployed to staging, then prompt you to push to production
```

Comment:
I would break this into a build, test, or CI/CD category as they are well-established practices in the industry.